manage.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
These imports should ideally be collected on the same line (same for the other places this happens)